Flick/Group/GroupSettingsViewController.swift

```diff
@@ -239,13 +282,27 @@ extension GroupSettingsViewController: UITableViewDataSource, UITableViewDelegat
 
 }
 
-extension GroupSettingsViewController: ModalDelegate, RenameGroupDelegate {
+extension GroupSettingsViewController: ModalDelegate, RenameGroupDelegate, AddMembersDelegate {
 
     func dismissModal(modalView: UIView) {
         modalView.removeFromSuperview()
     }
 
     func renameGroup(title: String) {
-        print("Rename group")
+        NetworkManager.updateGroup(id: group.id, name: title) { [weak self] group in
+            guard let self = self else { return }
+            DispatchQueue.main.async {
```

Comment:
can you also reuse `reloadGroupMembers` here?

Reply:
i think it might be a little confusing using `reloadGroupMembers` since it's not really reloading group members